[2024-03-09T17:02:46.438+0000] {scheduler_job_runner.py:781} ERROR - Executor reports task instance <TaskInstance: download_rocket_launches.get_pictures manual__2024-03-09T17:01:21.763730+00:00 [queued]> finished (failed) although the task says it's queued. (Info: None) Was the task killed externally?
